### DISCUSSION TITLE
possible to use phone as camera

### DISCUSSION THREAD

**Discussion by archef2000** [blakeblackshear/frigate#2700] [Ask A Question]
hello i currently have setup some old phones ( z.b. S5-mini ) streaming http/s video mjpeg the problem i cant figure out how to use this stream with frigate or do i need to use rtsp because then i would need to buy new cameras that are able to streamrtsp content

**Comment by rootiest** [comment]
You can use the TinyCam app on Android to create an RTMP h264 stream from your phone. It may be possible to make your mjpeg/http stream work (check the docs for MJPEG under camera-specific settings) but this will work much more efficiently as MJPEG streams have to be transcoded by the Frigate host.

**Reply by archef2000** [reply]
Hello thank you for your fast answer i have downloaded the app but i cant find the options to set this phone as camera or do i need to add my camera to this phone and then use this phone as source for frigate ?
